model/flow/header.go

```diff
@@ -43,6 +44,9 @@ type Header struct {
 
 	ProposerSigData []byte // signature of the proposer over the new block. Not a single cryptographic
 	// signature since the data represents cryptographic signatures serialized in some way (concatenation or other)
+
+	LastViewTC *TimeoutCertificate // Timeout certificate for previous view, it can be nil
+	// it has to be present if previous round ended with timeout.
```

Comment:
```suggestion
	// LastViewTC is the timeout certificate for previous view. It can be nil and
	// has to be present if previous round ended with timeout.
	LastViewTC *TimeoutCertificate 
```

Not a problem with this PR, just noticed it here: we aren't formatting these comments properly as GoDoc. Comments for a field shouldn't be below the field, otherwise they aren't associated with that field and don't show in documentation tools ([godoc site example](https://pkg.go.dev/github.com/onflow/flow-go/model/flow#Header)). 

For such an important model, I think it's worthwhile to fix all the fields here.